src/DotLiquid.Tests/Tags/ParamTest.cs

```diff
@@ -29,11 +29,12 @@ public void TestInitialize_SyntaxValidation()
         [TestCase("Syntax = 'DotLiquid21' | replace: '21', '10'")] // Filters is not valid in this tag
         public void TestInvalidOptions(string markup)
         {
-            var tag = new Param();
-            tag.Initialize(tagName: "param", markup: markup, tokens: null);
-
-            var context = new Context(new CultureInfo("en-US"));
-            Assert.Throws<SyntaxException>(() => tag.Render(context, new StringWriter()));
+            Assert.Throws<SyntaxException>(() => {
```

Comment:
Why this refactoring? The previous code was setting up the text context and only asserting on the specific call that is expected to throw.

Reply:
Good catch. I refactored because the exception is now thrown earlier in the process.